### PR TITLE
[documentation] Remove instructions about creating the support file

### PR DESCRIPTION
**Pull request by GeorgianaElena**
Fixes https://github.com/2i2c-org/infrastructure/issues/2187

### CODE REVIEW

File: docs/hub-deployment-guide/deploy-support/configure-support.md [hunk 1]
```diff
@@ -12,47 +12,16 @@ then a lot of these files will have already been created for you and you do not
 need to recreate them, only update them if required.
 ```
 
-## Create a `support.values.yaml` file in your chosen cluster folder
+## Make sure the `support.values.yaml` file in your chosen cluster folder is correctly configured
```
Comment (consideRatio) [inline docs/hub-deployment-guide/deploy-support/configure-support.md:15]:
```suggestion
## Make sure `support.values.yaml` is correctly configured
```

When things are very verbose, they can become harder to follow by increasing the amount of text to read and overview. The initial paragraph below clarifies the full path still, so i figure this is a good place to cut down on the verbosity.

File: docs/hub-deployment-guide/deploy-support/configure-support.md [hunk 2]
```diff
@@ -12,47 +12,16 @@ then a lot of these files will have already been created for you and you do not
...
-      hosts:
-        - <prometheus-domain>
-      tls:
-        - secretName: prometheus-tls
-          hosts:
-            - <prometheus-domain>
-```
+If you are deploying the support chart on an Azure cluster, you **must** manually create such a file using the template mentioned above.
+
+You must also set an annotation for `ingress-nginx`'s k8s Service resource.
```
Comment (consideRatio) [inline docs/hub-deployment-guide/deploy-support/configure-support.md:23]: Is this for Azure? It could be put in the same paragraph as the one above then

**Review by consideRatio — APPROVED** [review]
Wieeee nice!

**Comment by GeorgianaElena** [comment]
Thank you @consideRatio <3

**Pull request merged**